.gitIgnore

```diff
@@ -0,0 +1,127 @@
+# Logs
```

Comment:
nit: rename to `.gitignore` (vs `.gitIgnore`).

Reply:
Fixed in https://github.com/fiatconnect/specification/pull/41/commits/ec5283d4db8dee8497333ff8b0e2b60518bda332!